extensions/socketio/src/__tests__/fixtures/application.ts~HEAD

```diff
@@ -0,0 +1,37 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
```

Comment:
is there a particular usage of this file? It seems to be added accidentally because of some GitHub operations?

Reply:
yes, it seems so. I'll update this